beacon_chain/utils/bls/src/public_key.rs

```diff
@@ -20,6 +21,13 @@ impl PublicKey {
     }
 }
 
+impl default::Default for PublicKey {
+    fn default() -> Self {
+        let secret_key = SecretKey::random();
```

Comment:
Having `Default` be a random value is a bit odd, but I can't think of any better solution.

Random is safer than a known constant IMO.

I think this is fine, just mentioning it in case you have any other thoughts.

Reply:
yeah -- i've seen a lot of debate in the Rust community around the semantics of `Default`. 

i think what is here is fine because the main way this method is used is in testing. If a client is directly using `PublicKey::default()` then we could say the consequences are on them ;)

I'm open to considering other alternatives though because we generally want to provide "safe" software, which is most readily achieved by being heavy-handed in our tactics. (cf. borrowck).